File: 1000~1999/1065.py
def f(x):
    if x < 100:
        return 1

    arr = list(map(int, str(x)))
    if arr[2] - arr[1] == arr[1] - arr[0]:
        return 1

    return 0


cnt = 0
N = int(input())
for i in range(1, N+1):
    if f(i) == 1:
        cnt += 1
print(cnt)

File: c/1000~1999/1065.c
#include <stdio.h>

int f(int x)
{
	if (x < 100) return 1;

	char arr[3] = { x / 100, (x % 100) / 10, x % 10 };
	if (arr[1] - arr[0] == arr[2] - arr[1]) return 1;

	return 0;
}

int main()
{
	int n = 0;
	scanf("%d", &n);

	int cnt = 0;
	for (int i = 1; i <= n; i++) {
		if (f(i)) cnt++;
	}
	printf("%d", cnt);
	return 0;
}
